doc/users/index.rst

```diff
@@ -12,7 +12,7 @@ User's Guide
 .. toctree::
     :maxdepth: 2
 
-    intro.rst
+    history.rst
```

Comment:
I am not familiar with guide writing but it seems odd to me that user guide starts with a history chapter.

Reply:
I don't think many people read the docs in linear order tbh